<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidgetTab</class>
 <widget class="QWidget" name="QWidgetTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWidgetTab</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QWidgetTab{
	background-color: #ffffff
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">#tabWidget{
	background-color: #ffffff;
} 

QTabWidget::pane { /* The tab widget frame */
	top: 0px;
	border-top: 1px solid #dddddd;
	margin-left: 0px;
	margin-right: 0px;
	margin-bottom:0px;
}
/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	margin-top: 26px;
	margin-left: 2px;
    padding-left:0px;
	padding-right:0px;
	height:28px;
	width: 84px;
}

QTabBar::tab:selected {
	font: bold 11pt &quot;Microsoft YaHei&quot;;
	color: #006eff;
    background: #ffffff;
	border-bottom: 2px solid #006eff;
}

QTabBar::tab:!selected {
    margin-top: 26px; /* make non-selected tabs look smaller */
	font: 11pt &quot;Microsoft YaHei&quot;;
	color: #444444;
    background: #ffffff;
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_camera">
      <attribute name="title">
       <string>摄像头</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_camera" native="true">
         <property name="minimumSize">
          <size>
           <width>780</width>
           <height>642</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_board">
      <attribute name="title">
       <string>白板</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_board" native="true">
         <property name="minimumSize">
          <size>
           <width>780</width>
           <height>642</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_screen">
      <attribute name="title">
       <string>屏幕分享</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stacked_screen">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_screen_options">
          <property name="styleSheet">
           <string notr="true">#page_screen_options{
	background: #f2f2f2;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>486</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_area_share">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: 1px solid #dddddd;
	background:  #ffffff;
	color: #000000;
	font: 9pt &quot;Microsoft YaHei&quot;;
}

QPushButton:hover {
	border: 1px solid #d2d2d2;
	background:  #f2f2f2;
}

QPushButton:pressed {
	border: 1px solid #d2d2d2;
	background:  #e5e5e5;
}

QPushButton:disable {
	border: 1px solid #d5d5d5;
	color: #bbbbbb;
	background:  #f2f2f2;
}</string>
               </property>
               <property name="text">
                <string>局部屏幕分享</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_full_share">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	background:  #006eff;
	color: #ffffff;
	font: 9pt &quot;Microsoft YaHei&quot;;
}

QPushButton:hover {
	background:  #0068f2;
}

QPushButton:pressed {
	background:  #0062e5;
}

QPushButton:disable {
	border: 1px solid #d5d5d5;
	color: #bbbbbb;
	background:  #f2f2f2;
}</string>
               </property>
               <property name="text">
                <string>全屏分享</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>486</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_screen_content">
          <property name="styleSheet">
           <string notr="true">#page_screen_content{
	background: #f2f2f2;
}</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_media">
      <attribute name="title">
       <string>视频分享</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stacked_media">
         <widget class="QWidget" name="page_media_options">
          <property name="styleSheet">
           <string notr="true">#page_media_options{
	background: #f2f2f2;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_media_share">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	border: none;
	background:  #006eff;
	color: #ffffff;
	font: 9pt &quot;Microsoft YaHei&quot;;
}

QPushButton:hover {
	background:  #0068f2;
}

QPushButton:pressed {
	background:  #0062e5;
}

QPushButton:disable {
	border: 1px solid #d5d5d5;
	color: #bbbbbb;
	background:  #f2f2f2;
}</string>
                 </property>
                 <property name="text">
                  <string>选择视频文件</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #888888;
font: 9pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>支持avi、mp4格式的视频文件</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_media_content">
          <property name="styleSheet">
           <string notr="true">#page_media_content{
	background: #f2f2f2;
}</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
